framework/db/QueryBuilder.php

```diff
@@ -1733,4 +1733,18 @@ public function bindParam($value, &$params)
 
         return $phName;
     }
+
+    /**
+     * Extracts table alias if there is one or returns false
+     * @param $table
+     * @return bool|array
+     */
+    protected function extractAlias($table)
```

Comment:
Need "since" tag because method visibility is `protected` thus it can be used now.